Bio/EBI/__init__.py

```diff
@@ -0,0 +1,5 @@
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
```

Comment:
Can we have all of ``Bio.EBI`` dual licenced from the start please? See #898 and our current wording as used in ``Bio/SeqRecord.py``,

```python
# This file is part of the Biopython distribution and governed by your
# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
# Please see the LICENSE file that should have been included as part of this
# package.
```